Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/ConnectionClass.java

```diff
@@ -0,0 +1,28 @@
+package tddmicroexercises.telemetrysystem;
+import java.util.Random;
+
+public class ConnectionClass implements Connection{
```

Comment:
Lets not prefix or suffix the class name with "Class"
Rename it

Reply:
Renamed the class to COnnectionImpl